### ISSUE TITLE
Relocation error when running statically compiled pyinstaller on Debian

### ISSUE THREAD

**Issue by grandrew** [JonathonReinhart/staticx#94]
Compiled on Ubuntu 18.04 (4.15)

Running on Debian 9.9 (4.19)

```
/tmp/staticx-GDJODH/.staticx.prog: relocation error: /lib/x86_64-linux-gnu/libnss_dns.so.2: symbol __res_maybe_init version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```

```
ldd --version
ldd (Debian GLIBC 2.24-11+deb9u4) 2.24
```

and

```
ldd --version
ldd (Ubuntu GLIBC 2.27-3ubuntu1) 2.27
```

**Comment by grandrew** [comment]
Building on Ubuntu16 with same GLIBC version (2.24) make it work without relocation errors on Debian9.9 but leads to segmentation fault on Ubuntu 18 with GLIBC 2.27

Which is clearly an issue with GLIBC incompatibility probably caused by libraries like libnss and libdns that are not included in the "build"

**Comment by grandrew** [comment]
Found a problem and solution:

Python (at least v3.7) loads libraries libnss_dns.so and libresolv.so dynamically and only if certain calls are made by the actual python script, so staticx can not detect that they are used

the solution is to include the libraries explicitly:

```
staticx -l /lib/x86_64-linux-gnu/libnss_dns.so.2 -l /lib/x86_64-linux-gnu/libresolv.so.2 ...
```

**Comment by JonathonReinhart** [comment]
Hi @grandrew.

Version [`v0.11.0`](https://github.com/JonathonReinhart/staticx/releases/tag/v0.11.0) was just released which contains #139, a major fix for NSS. With this version, you shouldn't run into this type of issue any more, and you shouldn't need to pass those `-l` options.

I'm confident it will fix the problem you were seeing, so I'm going to close this issue.

Please try it out and report back! Thanks for the interest in my project.
